projects/canopy/src/lib/filter-buttons/filter-button/filter-button.component.html

```diff
@@ -0,0 +1,21 @@
+<input
+  (click)="onCheck()"
+  [attr.aria-describedby]="variant === 'select-multiple' ? ariaDescribedBy : null"
```

Comment:
Here as well. If `ariaDescribedBy` is undefined I would set it to `null`.

Reply:
Done.